### DISCUSSION TITLE
Add contextual menu to links if they are long clicked

### DISCUSSION THREAD

**Discussion by smed79** [spacecowboy/Feeder#357] [Ideas]
Hello,

in article links, we can only copy text or click them.

I suggest adding a contextual menu to links if they are **long clicked** with the below features: 

- Open in the Default browser (if the **Custom tab** is selected)
- Open in web view (if the **Default browser** is selected)
- Copy text
- Copy link
- Share link

Thanks in advance.

**Comment by spacecowboy** [comment]
Technical note, not sure if long-pressing links is supported in Compose
